lib/rubocop/cop/style/empty_else.rb

```diff
@@ -112,14 +112,24 @@ def nil_check(node)
 
         def autocorrect(node)
           return false if autocorrect_forbidden?(node.type.to_s)
+          return false if comment_in_else?(node)
 
           lambda do |corrector|
             end_pos = base_if_node(node).loc.end.begin_pos
-
             corrector.remove(range_between(node.loc.else.begin_pos, end_pos))
           end
         end
 
+        def comment_in_else?(node)
+          range = else_line_range(node.loc)
+          processed_source.comments.find { |c| range.include?(c.loc.line) }
```

Comment:
@Drenmi This looks like something common enough that we should extract it (checking there are comments in some range, maybe even something more granular).

Reply:
Agree.